index.js

```diff
@@ -59,14 +59,60 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
+const h1 = document.querySelector('h1')
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
 }
 
+h1.addEventListener('click', function(){
+    if(h1.style.color === 'red'){
+        h1.style.color = 'black'
+    }
+    else {
+        h1.style.color = 'red'
+    }
+})
```

Comment:
Awesome! We can also use the `toggleColor` function here! 


All we need to do is add an event listener to the header variable. We provide a callback function to be executed when our "click" event gets triggered.

The callback function we can pass in to the event listener is already made for us!. We just need to pass in the event target as an argument to the function.

```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})

```

With ES6 notation

```
header.addEventListener("click", e => toggleColor(e.target))
```